public/index.html

```diff
@@ -29,17 +29,17 @@ <h3><span id="country_claps"></span> in <span id="country_name"></span></h3>
         <img class="flagDivImg" src="./images/world.png"/>
         <h3><span id="global_claps"></span> Worldwide</h3>
       </div>
-      <a class="button">Share the CLAP!</a>
-      <a class="button js-link" href="#help">More ways to HELP</a>
+      <a class="button" style="width: 100%;">Share the CLAP!</a>
+      <a class="button js-link" href="#help" style="width: 100%;">More ways to HELP</a>
     </section>
     <!-- /RESULTS -->
 
     <!-- HELP -->
     <section id="help" style="display: none;">
       <h3>More ways in which you can help!</h3>
-      <a class="button js-link" href="#contact">Send an ecouragement message to the Doctors and Nurses in my area.</a>
-      <a class="button js-link" href="#contact">Donate resources to the cause.</a>
-      <a class="button js-link" href="#contact">Help/Join the ClapHelp team.</a>
+      <a class="button js-link" href="#contact" style="width: 100%;">Send an ecouragement message to the Doctors and Nurses in my area.</a>
+      <a class="button js-link" href="#contact" style="width: 100%;">Donate resources to the cause.</a>
+      <a class="button js-link" href="#contact" style="width: 100%;">Help/Join the ClapHelp team.</a>
```

Comment:
if many buttons require the same rules, it's best practice to create another class and add this style there. That would make the code DRY (Don't Repeat Yourself).
Example:
```html
<a class="button js-link responsive></a>
```
```css
.button.responsive {
  width: 100%;
}
```